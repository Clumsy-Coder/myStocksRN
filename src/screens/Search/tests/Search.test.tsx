import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';

import SearchScreen from 'src/screens/Search';

describe('<Search />', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<SearchScreen />);

    expect(wrapper).toMatchSnapshot();
  });
});
