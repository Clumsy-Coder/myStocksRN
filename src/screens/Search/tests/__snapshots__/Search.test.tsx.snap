// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> Renders correctly 1`] = `
<Styled(Container)>
  <Styled(Content)>
    <Connect(SearchBar) />
  </Styled(Content)>
</Styled(Container)>
`;
